pennylane/io.py

```diff
@@ -146,13 +156,99 @@ def circuit_loaded_qiskit_circuit():
         return load(quantum_circuit, format="qiskit", measurements=measurements)
     except ValueError as e:
         if e.args[0].split(".")[0] == "Converter does not exist":
-            raise RuntimeError(
-                "Conversion from Qiskit requires the PennyLane-Qiskit plugin. "
-                "You can install the plugin by running: pip install pennylane-qiskit. "
-                "You may need to restart your kernel or environment after installation. "
-                "If you have any difficulties, you can reach out on the PennyLane forum at "
-                "https://discuss.pennylane.ai/c/pennylane-plugins/pennylane-qiskit/"
-            ) from e
+            raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
+        raise e
+
+
+def from_qiskit_op(qiskit_op, params=None, wires=None):
+    """Loads Qiskit SparsePauliOp objects by using the converter in the PennyLane-Qiskit plugin.
+
+    Args:
+        qiskit_op (qiskit.quantum_info.SparsePauliOp): the SparsePauliOp to be converted
+        params (Any): optional assignment of coefficient values for the SparsePauliOp; see the
```

Comment:
Not quite: the type is either `None`, `Sequence`, or `Mapping`, [or more specifically](https://docs.quantum.ibm.com/api/qiskit/qiskit.quantum_info.SparsePauliOp#assign_parameters),
```python
None|Mapping[Parameter, complex|ParameterExpression] | Sequence[complex|ParameterExpression]
```
I opted to leave this as `Any` because we don't really care about the type (we just forward this argument to `assign_parameters()`), to keep things simple, and to insulate ourselves from future changes made by Qiskit.

Reply:
Thanks, got it! Ok `Any` is good.